client/src/pages/AvailabilitySettings.js

```diff
@@ -1,15 +1,62 @@
 import React from "react";
-import AvailabilityWidget from "../component/AvailabilityWidget"
+import history from "../history";
+import AvailabilityWidget from "../component/AvailabilityWidget";
 
 const AvailabilitySettings = () => {
-    let heading = "Set your availability";
-    return (
-        <React.Fragment>
-          <AvailabilityWidget
-            heading={heading}
-          />
-        </React.Fragment>
-      );
+  const convertToUtc = (time) => {
+    const now = new Date();
+    let d = new Date(
+      now.getFullYear(),
+      now.getMonth(),
+      now.getDate(),
+      time.slice(0, 2),
+      time.slice(3),
+      0
+    );
+    let hours = d.getUTCHours();
+    let mintue = d.getUTCMinutes();
+    if (hours < 10) {
+      hours = "0" + hours;
+    }
+    if (mintue < 10) {
+      mintue = "0" + mintue;
+    }
+    return `${hours}:${mintue}`;
+  };
+  const handleFinish = (start, end, daysSelected) => {
+    const unique_url = localStorage.getItem("unique_url");

```

Comment:
do we really need localstorage for this? these should be coming from the backend

Reply:
yes. I am planning to use context to remove local storage issue.